style.css

```diff
@@ -0,0 +1,35 @@
+
+*{
+    text-decoration: none;
+    color: black;
+}
+
+
+header{
+    background-color: #fff;
+    border-bottom: 1px solid #eee;
+    padding: 20px 0;
+    margin: 10px auto;
+    align-items: center;
+    text-align: center;
```

Comment:
you have to re-use one thing function  twice

Reply:
- [ ] Kindly I recommend you to phrased the comments politely, as suggestions instead of orders and  includes an explanation, a "why", for each requested change with comments are clear and easy to understand, Here comments you suggested are not clear and easy to understand.